src/main/java/seedu/task/logic/commands/EditEventCommand.java

```diff
@@ -119,9 +119,6 @@ public CommandResult undo() {
 			return new CommandResult(String.format(MESSAGE_EDIT_EVENT_SUCCESS, editEvent));
 		} catch (UniqueEventList.DuplicateEventException e) {
 			return new CommandResult(MESSAGE_DUPLICATE_EVENT);
-		} catch (IndexOutOfBoundsException ie) {
-			indicateAttemptToExecuteIncorrectCommand();
-			return new CommandResult(Messages.MESSAGE_INVALID_EVENT_DISPLAYED_INDEX);
 		}
```

Comment:
It won't when executing the undo command. The fact that the edit command was successful means that the index of the new edited event will be within bounds.